engine/table/src/main/java/io/deephaven/engine/table/impl/select/python/ArgumentsChunked.java

```diff
@@ -3,22 +3,54 @@
  */
 package io.deephaven.engine.table.impl.select.python;
 
+import io.deephaven.base.Pair;
 import io.deephaven.chunk.attributes.Any;
 import io.deephaven.chunk.*;
+import io.deephaven.engine.util.PythonScopeJpyImpl;
+import io.deephaven.engine.util.PythonScopeJpyImpl.CallableWrapper;
 import io.deephaven.util.PrimitiveArrayType;
 
+import java.util.ArrayList;
 import java.util.Objects;
 
 class ArgumentsChunked {
-    static ArgumentsChunked buildArguments(Chunk<?>[] __sources) {
+    // static ArgumentsChunked buildArguments(Chunk<?>[] __sources) {
+    // final Class<?>[] paramTypes = new Class[__sources.length];
+    // final Object[] params = new Object[__sources.length];
+    // for (int i = 0; i < __sources.length; i++) {
+    // final ChunkToArray<?> cta = __sources[i].walk(new ChunkToArray<>());
+    // paramTypes[i] = Objects.requireNonNull(cta.getArrayType());
+    // params[i] = Objects.requireNonNull(cta.getArray());
+    // }
+    // return new ArgumentsChunked(paramTypes, params);
+    // }
+
+    static ArgumentsChunked buildArguments(Chunk<?>[] __sources, CallableWrapper callableWrapper) {
         final Class<?>[] paramTypes = new Class[__sources.length];
         final Object[] params = new Object[__sources.length];
         for (int i = 0; i < __sources.length; i++) {
             final ChunkToArray<?> cta = __sources[i].walk(new ChunkToArray<>());
             paramTypes[i] = Objects.requireNonNull(cta.getArrayType());
             params[i] = Objects.requireNonNull(cta.getArray());
         }
-        return new ArgumentsChunked(paramTypes, params);
+
+        ArrayList<Pair<?, ?>> args = callableWrapper.getArgs();
+        final Object[] chunkedArgs = new Object[args.size()];
+        Class<?>[] argTypes = callableWrapper.getArgTypes();
+        Class<?>[] chunkedArgTypes = new Class[argTypes.length];
```

Comment:
Shouldn't we be able to allocate this a single time? Are we making a new `ArgumentsChunked` per invocation?